### ISSUE TITLE
Panic while typing a filename in image()

### ISSUE THREAD

**Issue by classabbyamp** [nvarner/typst-lsp#92]
- **Component**: 
    - [x] VSCode Extension (used with local lsp binary)
    - [ ] VSCodium Extension
    - [ ] LSP (used with other editor)
- **Extension version**: 0.4.0
- **LSP version**: 0.4.0
- **OS version and name**: Void Linux

- [x] I am on the [latest](https://github.com/nvarner/typst-lsp/tags) stable version of the extension/LSP.
- [x] I have searched the [issues](https://github.com/nvarner/typst-lsp/issues) of this repo and believe that this is not a duplicate.

## Issue

lsp server panics while typing the path of an `image()`

### Logs

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/lsp_typst_boundary/world.rs:62:61
```
five times then a message from vscode saying it wouldn't restart the server

**Comment by nvarner** [comment]
I'm working on this bug now. It should be fixed soon.

**Comment by nvarner** [comment]
This should be fixed in master and the new v0.4.1 release. Please reopen if the issue persists.
